polling_stations/apps/file_uploads/templates/file_uploads/council_detail.html

```diff
@@ -46,14 +46,11 @@ <h2>Uploads</h2>
                             {% endif %}
                         </td>
                         <td>
-                            {{ upload.status }}
-                            {% if upload.status == "Pending" %}
-                                <br><small>(Refresh page to update)</small>
-                            {% endif %}
```

Comment:
I think it would be nice to keep the note about refreshing the page - but check for `Waiting` as well as `Pending`.